.phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Genesis Featured Widget Amplified">
+    <description>The code standard for the Genesis Featured Widget Amplified plugin.</description>
+
+    <!-- What to scan -->
+    <file>.</file>
+    <exclude-pattern>vendor/</exclude-pattern>
```

Comment:
Need to keep some of the exclude patterns. WPCS at the min.

Reply:
The exclusions that have been removed, have no relevance to this repo - no `style.css`, no `node_modules/`, no `tests/`, no minified files or gulp files etc.

If these are added, then of course, add them back in here.

The `wpcs/` is typically found when using the old way (`create-project`) of running WPCS in Travis, and that's no longer needed. If you do need CI, just get it to run `vendor/bin/phpcs`, the same as you would locally.

Because WPCS (and PHPCS, and PHPCompatibility) are marked as dependencies in `composer.json`, then running `composer install` is all that's needed - you don't need to pull in WPCS as an extra task (which then requires the `create-project` which makes use of the `wpcs/` directory).

Reply:
Cool. I need to change how I’m pulling WPCS then but I like this. Thanks.